bridges/snowbridge/pallets/ethereum-client/src/lib.rs

```diff
@@ -395,6 +397,8 @@ pub mod pallet {
 					),
 					Error::<T>::InvalidSyncCommitteeMerkleProof
 				);
+			} else {
+				ensure!(update_finalized_period == store_period, <Error<T>>::SyncCommitteeUpdateRequired);
 			}
```

Comment:
Prefer to use `signature_period == store_period`, same as 
https://github.com/Snowfork/polkadot-sdk/blob/504d0e6d42daa15854ea75d380323c5ecf33113b/bridges/snowbridge/pallets/ethereum-client/src/lib.rs#L319

Reply:
The check is not quite the same then. Also, could it not mean we not be able to import finalized headers at the end of an sync committee period where the signature period is in the next period but the finalized header is in the current period? For example:
finalized_slot = 9076704 = period 1107
signature_slot = 9076769 = period 1108